wolfssl/internal.h

```diff
@@ -3449,8 +3473,8 @@ struct WOLFSSL_CTX {
     DerBuffer*  certificate;
     DerBuffer*  certChain;
                  /* chain after self, in DER, with leading size for each cert */
-    #if defined(OPENSSL_EXTRA) || defined(WOLFSSL_EXTRA) || defined(HAVE_LIGHTY)
-    WOLF_STACK_OF(WOLFSSL_X509_NAME)* ca_names;
+    #ifndef WOLFSSL_NO_CA_NAMES
+    WOLF_STACK_OF(WOLFSSL_X509_NAME)* client_ca_names;
```

Comment:
Why did the name have to change?
Customers may be using this field directly and we can't be changing it on them.

Reply:
I changed the name because in OpenSSL what we called `ca_names` corresponded to what OpenSSL calls `client_ca_names`. I think that it may be confusing in the future if we decide to implement what OpenSSL calls `ca_names`.

`client_ca_names` is the list of names that the server wants to send to the client. So it is the list of client names that the server will accept.

This change is inside of `WOLFSSL` and `WOLFSSL_CTX`. Both are in `internal.h`. This is not a public header. Are you sure that this is something we want to support for customers? If a customer really is using this field, then it would be an easy fix for them to change `ca_names` -> `client_ca_names`.